src/apps/KintoApp.sol

```diff
@@ -0,0 +1,278 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.12;
+
+import "@openzeppelin/contracts-upgradeable/token/ERC721/ERC721Upgradeable.sol";
+import "@openzeppelin/contracts-upgradeable/token/ERC721/extensions/ERC721EnumerableUpgradeable.sol";
```

Comment:
What do we need `ERC721Enumerable` for?

Reply:
to get all the apps  created by a specific dev

Reply:
makes sense